src/Eventuras.Domain/ExternalCourse.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Eventuras.Domain
+{
+    public class ExternalCourse
```

Comment:
Name this `ExternalRegistrationServices` as it might later be used for video events, and for example zoom registrations?